crypto/x509util/profile.go

```diff
@@ -21,6 +22,38 @@ import (
 	"github.com/smallstep/cli/utils"
 )
 
+var (
+	oidExtSubjectKeyID          = []int{2, 5, 29, 14}
+	oidExtKeyUsage              = []int{2, 5, 29, 15}
+	oidExtExtendedKeyUsage      = []int{2, 5, 29, 37}
+	oidExtAuthorityKeyID        = []int{2, 5, 29, 35}
+	oidExtBasicConstraints      = []int{2, 5, 29, 19}
+	oidExtSubjectAltName        = []int{2, 5, 29, 17}
+	oidExtCertificatePolicies   = []int{2, 5, 29, 32}
+	oidExtNameConstraints       = []int{2, 5, 29, 30}
+	oidExtCRLDistributionPoints = []int{2, 5, 29, 31}
+	oidExtAuthorityInfoAccess   = []int{1, 3, 6, 1, 5, 5, 7, 1, 1}
+	oidStandardExtHashMap       = map[string]int{}
```

Comment:
Not super important, but you should use `map[string]stuct{}` for "sets" as it takes less memory.